src/test/java/rife/tools/TestFileUtils.java

```diff
@@ -891,25 +875,25 @@ public void testGetJavaFileListNullDirectory() {
     @Test
     void testPermissionsFromMode() {
         assertTrue(permissionsFromMode(0000).isEmpty());
-        assertTrue(permissionsFromMode(0001).containsAll(Set.of(OTHERS_EXECUTE)));
```

Comment:
I would prefer for this to remain `containsAll` throughout this test since it clearly shows the pattern of checks.